### PR TITLE
Dts 36748 code shift

### PR DESCRIPTION
**Pull request by aksshriv1**

### CODE REVIEW

File: customapi/src/main/java/com/publicissapient/kpidashboard/apis/jira/service/releasedashboard/JiraReleaseServiceR.java [hunk 1]
```diff
@@ -306,6 +310,45 @@ public List<String> getReleaseList() {
...
+	 * Get Sum of StoryPoint for List of JiraIssue
+	 *
+	 * @param jiraIssueList
+	 *            List<JiraIssue>
+	 * @param fieldMapping
+	 *            fieldMapping
+	 * @return Sum of Story Point
+	 */
+	public double getTicketEstimate(List<JiraIssue> jiraIssueList, FieldMapping fieldMapping, double ticketEstimate) {
+		if (CollectionUtils.isNotEmpty(jiraIssueList)) {
```
Comment (purushottam08) [inline customapi/src/main/java/com/publicissapient/kpidashboard/apis/jira/service/releasedashboard/JiraReleaseServiceR.java:336]: this function should also be moved to helper

File: customapi/src/main/java/com/publicissapient/kpidashboard/apis/jira/service/releasedashboard/JiraReleaseKPIService.java [hunk 2]
```diff
@@ -101,4 +101,31 @@ public List<JiraIssueCustomHistory> getJiraIssuesCustomHistoryFromBaseClass() {
...
 
+	/**
+	 * 
+	 * @param fieldMapping
+	 *            fieldMapping
+	 * @param releaseNameList
+	 *            releaseNameList
+	 * @return total jira issues list of selected closed releases
+	 */
+	public List<JiraIssue> getClosedReleaseJiraIssueList(FieldMapping fieldMapping, List<String> releaseNameList) {
```
Comment (purushottam08) [inline customapi/src/main/java/com/publicissapient/kpidashboard/apis/jira/service/releasedashboard/JiraReleaseKPIService.java:112]: remove this function as well

File: customapi/src/main/java/com/publicissapient/kpidashboard/apis/jira/service/releasedashboard/JiraReleaseServiceR.java (hunk 1 above)
Comment (aksshriv1) [reply customapi/src/main/java/com/publicissapient/kpidashboard/apis/jira/service/releasedashboard/JiraReleaseServiceR.java]: This function is moved to kpi level as it is used in this class only. In future if anyone want same code they can move it in helper. Currently not required

File: customapi/src/main/java/com/publicissapient/kpidashboard/apis/jira/service/releasedashboard/JiraReleaseKPIService.java (hunk 2 above)
Comment (aksshriv1) [reply customapi/src/main/java/com/publicissapient/kpidashboard/apis/jira/service/releasedashboard/JiraReleaseKPIService.java:112]: why?

File: customapi/src/main/java/com/publicissapient/kpidashboard/apis/jira/service/releasedashboard/JiraReleaseKPIService.java [hunk 3]
```diff
@@ -101,4 +101,31 @@ public List<JiraIssueCustomHistory> getJiraIssuesCustomHistoryFromBaseClass() {
...
+	 * Get Sum of StoryPoint for List of JiraIssue
+	 *
+	 * @param jiraIssueList
+	 *            List<JiraIssue>
+	 * @param fieldMapping
+	 *            fieldMapping
+	 * @return Sum of Story Points
+	 */
+	public Double getStoryPoint(List<JiraIssue> jiraIssueList, FieldMapping fieldMapping) {
+		double ticketEstimate = 0.0d;
```
[the remark this answers is not in the document]
Comment (aksshriv1) [reply customapi/src/main/java/com/publicissapient/kpidashboard/apis/jira/service/releasedashboard/JiraReleaseKPIService.java]: moved to kpi level

**Review by brahmanand1 — APPROVED** [review]

**Review by hirbabar — APPROVED** [review]

**Review by purushottam08 — APPROVED** [review]

**Pull request merged**